sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -4555,20 +4557,20 @@ var setParentClass = function(child, parent) {
         return 'modf';
     };
 
-    glsBuiltinPrecisionTests.Modf.prototype.doApply = function(ctx, iargs) {
+    glsBuiltinPrecisionTests.Modf.prototype.doApply = function(ctx, iargs, variablenames) {
         var intPart;
         var func1 = function(x) {
-            intPart = Math.floor(x);
+            intPart = Math.trunc(x);
             return x - intPart;
         };
         var func2 = function(x) {
-            return Math.floor(x);
+            return Math.trunc(x);
         };
 
         var fracIV = tcuInterval.applyMonotone1p(func1, iargs.a);
         var wholeIV = tcuInterval.applyMonotone1p(func2, iargs.a);
 
-        iargs.b = wholeIV;
+        ctx.env.m_map[variablenames[1]] = wholeIV;
```

Comment:
A question: if you always only use one variable name, why do you need to pass four in?